hack/windows/Prepare-Node.ps1

```diff
@@ -71,8 +91,15 @@ DownloadFile "$global:KubernetesPath\kubeadm.exe" "https://$KubernetesURL/$Kuber
 DownloadFile "$global:KubernetesPath\wins.exe" https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
 
 # Create host network to allow kubelet to schedule hostNetwork pods
-Write-Host "Creating Docker host network"
-docker network create -d nat host
+if ($ContainerRuntime -eq "Docker") {
+    Write-Host "Creating Docker host network"
+    docker network create -d nat host
+} elseif ($ContainerRuntime -eq "ContainerD") {
+    Write-Host "Creating ContainerD host network"
+    DownloadFile "c:\k\hns.psm1" https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1
```

Comment:
May I know why you add containerd host network? We add docker hostnetwork because we need use it to build image on windows node. But in containerd cluster we have another docker testbed to build image.

Reply:
removed this part

Reply:
has this been addressed?